packages/react/src/internal/components/Alert.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import {Announce, type AnnounceProps} from './Announce'
+
+export type AlertProps = AnnounceProps
+
+export function Alert({children, ...rest}: AlertProps) {
+  return (
+    <Announce {...rest} politeness="assertive">
```

Comment:
(not blocking) This component feels pretty minimal, should we reduce just 1 Announce component where politeness can be a required prop?

Reply:
The big motivator for `Alert` and `Status` was that they correspond to the equivalent roles that people tend to look for. I almost would want `Announce` as an implementation detail in case we discover that we can implement `Alert` and `Status` differently in the future lol.

I'd love to keep them as a result, and encourage their use over `Announce`, since it'd be more clear what `Alert` or `Status` would be internally than `Announce`, especially to new contributors. It'd also let us swap out semantics in case we want to switch them up in the future

Reply:
Interesting, makes sense!